pkg/apps/gitops.go

```diff
@@ -42,6 +42,16 @@ func (o *GitOpsOptions) AddApp(app string, dir string, version string, repositor
 		GitProvider: o.GitProvider,
 	}
 
+	prDir := o.EnvironmentsDir
+	if prDir == "" {
```

Comment:
This confuses me. What's happening here. So first you set the `prDir` to the `EnvironmentsDir` which I believe is `~/.jx/environments`, right? If it does not exist you create a tmp directory which eventually gets deleted. I get that part, but what if `~/.jx/environments` already exists?

Reply:
hmm, on second thought, I might be looking at something here which is work in progress, since `prDir` is not even used.

Reply:
Fixed that.